examples/USGS/USGS_NLCD_RELEASES_2019_REL_RCMAP_V5_TRENDS_YEAR.js

```diff
@@ -0,0 +1,36 @@
+// Import the NLCD RCMAP TRENDS YEAR collection.
+var dataset = ee.ImageCollection('projects/ee-rcmap/assets/RCMAP_V5_TRENDS/YEAR');
+
+// Filter the collection to the 2019 product.
+var nlcd2019 = dataset.filter(ee.Filter.eq('system:index', '2019')).first();

```

Comment:
Use `ee.Filter.date`.  Make sure you test this as I haven't.

```
var year = '2019';
var nlcd2019 = dataset.filter(ee.Filter.date(ee.Date(year).getRange('year')));
```

e.g.

https://github.com/google/earthengine-catalog/blob/main/examples/NOAA/NOAA_NHC_HURDAT2_atlantic.js#L5

Reply:
Updated, tested and worked